KenBurns/Classes/KenBurns.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 import UIKit
 import CalmParametricAnimations
 import SDWebImage
+import Kingfisher
```

Comment:
Don't we need to add this to Podspec here? https://github.com/calm/KenBurns/blob/b71ad6d0e986fa24d303c94ba5928f034ad47ef8/KenBurns.podspec#L15

Reply:
👍🏼 Added